package.json

```diff
@@ -1,11 +1,13 @@
 {
     "name": "graphql-gateway",
     "version": "1.0.0",
-    "description": "",
-    "main": "index.js",
+    "description": "A developer portal for GraphQL-Gate providing charts of allowed/blocked queries and their data as well as a multitude of configurable rate limiters to choose from.",
+    "main": "build/bundle.js",
```

Comment:
Is main the entry point to the server? Should it be index.ts?

Reply:
I'm unsure if main should point to the front end index or back end, here I was assuming front end.

Reply:
maybe someone else can comment on this.